src/flow/cadence/FLOAT.cdc

```diff
@@ -442,7 +441,7 @@ access(all) contract FLOAT: NonFungibleToken, ViewResolver {
         // A function every verifier must implement. 
         // Will have `assert`s in it to make sure
         // the user fits some criteria.
-        access(account) fun verify(_ params: {String: AnyStruct})
+        access(all) fun verify(_ params: {String: AnyStruct})

```

Comment:
@jacob-tucker is it the correct auth change here? Why was it made account-access previously?